src/App.tsx

```diff
@@ -1,61 +1,119 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+function getUser(userId: number): User | null {
```

Comment:
```suggestion
function getUserById(userId: number): User | null {
```
If you are searching some entity by some filter, specify it in function/method name